library/core/src/intrinsics.rs

```diff
@@ -1849,6 +1849,7 @@ pub(crate) fn is_nonoverlapping<T>(src: *const T, dst: *const T, count: usize) -
 #[inline]
 pub const unsafe fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: usize) {
     extern "rust-intrinsic" {
+        #[rustc_const_unstable(feature = "const_intrinsic_copy", issue = "none")]
```

Comment:
Looks like we have another case of this a few lines down, around line 1940, according to CI.

Reply:
yea, I had to rebase first ^^